From 2b506accc4c1e5775d5f50c7f72be7513485bf47 Mon Sep 17 00:00:00 2001
From: Alison Wang <b18965@freescale.com>
Date: Tue, 18 Apr 2017 13:50:38 +0800
Subject: [PATCH 319/767] ARM: configs: Add multi_v7_lpae.config to support
 ARM_LPAE

Add multi_v7_lpae.config to support the Large Physical Address
Extension.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/configs/multi_v7_lpae.config | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 arch/arm/configs/multi_v7_lpae.config

diff --git a/arch/arm/configs/multi_v7_lpae.config b/arch/arm/configs/multi_v7_lpae.config
new file mode 100644
index 000000000000..cd13d2df5fd5
--- /dev/null
+++ b/arch/arm/configs/multi_v7_lpae.config
@@ -0,0 +1,2 @@
+# Support for the Large Physical Address Extension
+CONFIG_ARM_LPAE=y
-- 
2.17.0

